components/activities/create-activity-dialog/create-activity-dialog.scss.js

```diff
@@ -0,0 +1,63 @@
+import { scss } from "../../../deps.ts";
+
+export default scss`
+.c-create-activity-dialog {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex: 1;
+  height: 100%;
+
+  .body {
+    padding: 0 24px;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
we should probably figure out a good standardization of when to use grid vs flex

Reply:
yeah I agree on needing to standardize, any ideas?

Reply:
the general consensus on line is anything one-directional use flex, 2d use grid